app/helpers/response_generation.py

```diff
@@ -0,0 +1,12 @@
+from flask import make_response, jsonify
+
+
+def make_error_msg(code, msg):
+    return make_response(jsonify({'success': False, 'error': {'code': code, 'message': msg}}), code)
+
+
+def create_response(base_url, short_id, error_message=None):
+    response = {"shorturl": base_url + short_id}
+    if error_message is not None:
+        response["error"] = error_message
+    return response
```

Comment:
Same as above. In the current viewer, the json returned when the url parameter is too long is "https://s.geo.admin.ch/toolong" and the viewer expects no error. After discussing with Chris, we decided that we would have a json which is compatible both way in this case: it contains both an error, and the 'shorturl'. This is the only case where we create an error message and it will be discontinued when we discard the old viewer.
So, same as above: if this goes live only when the old viewer is discarded, I'm getting rid of it. 
@boecklic , an opinon on this ?

Reply:
@ltkum no need to support the old viewer, old viewer uses mf-chsdi3 for as long as it lives, once we switch viewer, we also switch 'internal' services, so new viewer should only use new services